mlos_bench/mlos_bench/config/service/remote/azure/service-provision-vm.jsonc

```diff
@@ -4,7 +4,7 @@
 
         "config": {
 
-            "deployTemplatePath": "azure/arm-templates/azuredeploy-ubuntu-vm.jsonc",
+            "deployTemplatePath": "service/remote/azure/vm/arm-templates/azuredeploy-ubuntu-vm.jsonc",
```

Comment:
Here's another ergonomic fixup that'd be cool: make this path relative to this file so we can do 
```json
            "deployTemplatePath": "./arm-templates/azuredeploy-ubuntu-vm.jsonc",
```
instead.

Right now all path resolution is relative to the search paths so we have to at least use relative paths to that.

Reply:
Don't think I will try and tackle that here though.

Reply:
agree. I've been thinking about relative paths but also decided to put it off for a while. Let's live with what we have now and can focus on running the experiments